README.md

```diff
@@ -115,7 +115,7 @@ Read the [contributing section](CONTRIBUTING.md) for how to develop new features
 ### Why do we need a command line tool?
 
 This is a great question and one that we will ask ourselves for each new web-ext
-feature. Most web extension functionality is baked into the browsers
+feature. Most WebExtensions functionality is baked into the browsers
```

Comment:
since this relates to the API functionality the change is good but I think it would read less awkwardly in singular form: "WebExtension functionality..."